command_line/dials_import.py

```diff
@@ -571,9 +571,19 @@ def __call__(self, imageset_list):
                 if imageset.get_scan().is_still():
                     # make lots of experiments all pointing at one
                     # image set
-                    start, end = imageset.get_scan().get_array_range()
+
+                    # check if user has overridden the input - if yes, recall
+                    # that these are in people numbers (1...) and are inclusive
+                    if self.params.geometry.scan.image_range is not None:
+                        ustart, uend = self.params.geometry.scan.image_range
```

Comment:
```suggestion
                        user_start, user_end = self.params.geometry.scan.image_range
```
I assume "u" is short for "user" here? Might be worth spelling this out long hand to save future me the trouble of trying to figure out "what on earth does 'u' stand for?".